.github/workflows/check-labels.yml

```diff
@@ -0,0 +1,15 @@
+name: check-labels
+on:
+  pull_request:
+    types: [opened, reopened, labeled, unlabeled, synchronize]
```

Comment:
this might not need "synchronize" anymore

Reply:
Yepp. Not necessary to run on push IMHO.

Reply:
:+1:

(squashed right in)